### ISSUE TITLE
Add aria tags

### ISSUE THREAD

**Issue by Nat-Faeeria** [pojome/elementor#290]
It would be really usefull if we could add aria tags to the elements =)

**Comment by arielk** [comment]
What kind of Aria tags you expect? Have a specific example?

**Comment by Nat-Faeeria** [comment]
I was thinking of having a field in the "Advanced" section of each element, kind of like the "css classes" field, in order for developers to add aria tags

**Comment by ramiy** [comment]
@arielk, It can be a separate addon that adds accessibility capabilities to elementor. Or, each widget can have it's own accessibility fields in the "Advanced" section.

For example, "aria-hidden", "aria-label", "aria-labelledby", "role-presentation", or any other tag to help voice over software.

You should have an internal accessibility expert to review all your widgets and add "Advanced" fields for each element.

It can actually be a "Game Changer" for your product. Other builders are usually don't have builtin accessibility capabilities. And it's not so hard to implement this.

**Comment by arielk** [comment]
@ramiy your right, add-on could be a better way to do it in Elementor

**Comment by ramiy** [comment]
@arielk I'm not sure an add-on is the right solution. Each widget has it's unique components, it should be defined in the individual widget level. Not a general addon.

**Comment by ramiy** [comment]
@Nat-Faeeria  see #4030

**Comment by black-eye** [comment]
+1

**Comment by macedvisioned** [comment]
And also the form id needs a unique element

**Comment by new-dash** [comment]
If you only have a couple of tags to add why not do some DOM manipulation. Enqueue the script in a child theme and register it on the page id.  

You could also try [this plug-in](https://make.wordpress.org/accessibility/handbook/which-tools-can-i-use/wp-accessibility-plugin/)
